compiler-rt/lib/sanitizer_common/sanitizer_linux.cpp

```diff
@@ -181,9 +178,7 @@ static void KeepUnblocked(__sanitizer_sigset_t &newset,
 void BlockSignals(__sanitizer_sigset_t *oldset) {
   __sanitizer_sigset_t currentset;
 #  if !SANITIZER_ANDROID
-  // FIXME: SetSigProcMask and pthread_sigmask cause mysterious failures
-  // See also: comment in KeepUnblocked().
-  // In the meantime, we prefer to sometimes incorrectly unblock signals.
+  // FIXME: SetSigProcMask cause mysterious failures on Android
```

Comment:
can we create some reference? a bug in llvm or google/sanitizers repo that has the logs of the failing buildbot run?

Reply:
Filed https://github.com/google/sanitizers/issues/1816 and updated the FIXMEs